redash/templates/login.html

```diff
@@ -21,17 +21,18 @@
     {% endif %}
 
     {% if show_saml_login %}
-      <a href="/saml/login" class="login-button btn btn-default btn-block">SAML Login</a>
+      <a href="{{ url_for('saml_auth.sp_initiated', org_slug=org_slug, next=next) }}" class="login-button btn btn-default btn-block">SAML Login</a>
```

Comment:
@arikfr [Here](https://github.com/getredash/redash/blob/master/redash/templates/invite.html#L24-L41) is similar code - should it also be fixed?

Reply:
Good catch! You're of course correct. I'll fix it (and also grep the code for more places).